src/runtime/string.go

```diff
@@ -283,3 +283,15 @@ func cgo_GoBytes(ptr unsafe.Pointer, length uintptr) []byte {
 	}
 	return buf
 }
+
+func cgo_CBytes(b []byte) unsafe.Pointer {
+	p := malloc(uintptr(len(b)))
+	sliceHeader := struct {
+		p   unsafe.Pointer
+		len int
+		cap int
+	}{p, len(b), len(b)}
```

Comment:
TinyGo uses `uintptr` instead of `int` here. It's a small difference necessary to support AVR.